medicines/doc-index-updater/src/main.rs

```diff
@@ -12,29 +14,40 @@ async fn main() -> Result<(), Box<dyn error::Error>> {
     }
     pretty_env_logger::init();
 
-    let addr = format!(
-        "0.0.0.0:{}",
-        env::var("PORT").unwrap_or_else(|e| {
-            eprintln!("defaulting PORT to {} ({})", PORT, e);
-            PORT.to_string()
-        })
-    )
-    .parse::<SocketAddr>()?;
+    let addr = format!("0.0.0.0:{}", get_env_or_default("PORT", PORT.to_string()))
+        .parse::<SocketAddr>()?;
+
+    let redis_addr = get_env_or_default("REDIS_ADDR", "redis://127.0.0.1:6379/".to_string());
```

Comment:
is port `6379` is the default of Redis? If is not can we parametised it?

Reply:
Yep it's the default, for local use. The endpoint is already parameterized so I think this is OK